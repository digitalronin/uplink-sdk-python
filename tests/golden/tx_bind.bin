00000000: 03 ED 37 6D 52 35 64 37  73 36 63 4B 42 34 71 6A  ..7mR5d7s6cKB4qj
00000010: 75 58 31 6B 69 77 77 4E  74 79 67 66 55 52 68 46  uX1kiwwNtygfURhF
00000020: 51 39 34 33 57 52 78 4D  4E 63 6E 59 67 5A 46 63  Q943WRxMNcnYgZFc
00000030: 45 33 36 69 6F 68 71 72  58 4B 51 64 61 6A 55 64  E36iohqrXKQdajUd
00000040: 41 78 41 45 34 78 4D 54  55 78 4D 7A 59 34 4D 44  AxAE4xMTUxMzY4MD
00000050: 41 34 4D 6A 41 30 4E 54  59 34 4D 7A 4D 33 4D 7A  A4MjA0NTY4MzM3Mz
00000060: 63 35 4F 54 51 78 4D 6A  59 33 4E 7A 45 7A 4F 44  c5OTQxMjY3NzEzOD
00000070: 59 77 4D 54 55 77 4D 6A  59 79 4F 44 63 77 4F 54  YwMTUwMjYyODcwOT
00000080: 55 77 4D 7A 51 32 4D 6A  55 32 4D 6A 4D 32 4E 44  UwMzQ2MjU2MjM2ND
00000090: 51 78 4F 44 59 79 4E 7A  67 78 4D 44 67 35 4D 6A  QxODYyNzgxMDg5Mj
000000A0: 59 32 4F 54 41 33 4E 7A  6B 31 4E 6A 63 36 41 45  Y2OTA3Nzk1Njc6AE
000000B0: 30 35 4F 44 49 30 4E 54  49 34 4D 44 55 79 4D 6A  05ODI0NTI4MDUyMj
000000C0: 41 77 4D 7A 55 77 4E 54  59 30 4E 44 63 35 4E 7A  AwMzUwNTY0NDc5Nz
000000D0: 59 33 4D 44 67 30 4D 7A  45 77 4E 7A 49 33 4E 6A  Y3MDg0MzEwNzI3Nj
000000E0: 45 7A 4D 6A 59 77 4D 6A  41 31 4D 44 45 7A 4D 7A  EzMjYwMjA1MDEzMz
000000F0: 41 34 4D 6A 59 79 4E 54  63 32 4F 44 63 77 4E 6A  A4MjYyNTc2ODcwNj
00000100: 51 35 4D 54 59 77 4D 6A  67 33 4E 54 63 79 4E 54  Q5MTYwMjg3NTcyNT
00000110: 63 34 4F 44 51 32 4E 77  3D 3D                    c4ODQ2Nw==