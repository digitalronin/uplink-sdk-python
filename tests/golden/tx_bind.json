{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxAsset", 
        "contents": {
            "tag": "Bind", 
            "contents": {
                "asset": "43WRxMNcnYgZFcE36iohqrXKQdajUdAxeSn9mzE1ZedB", 
                "contract": "7mR5d7s6cKB4qjuX1kiwwNtygfURhFQ9TKvEd9kmq6QL", 
                "proof": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE05ODI0NTI4MDUyMjAwMzUwNTY0NDc5NzY3MDg0MzEwNzI3NjEzMjYwMjA1MDEzMzA4MjYyNTc2ODcwNjQ5MTYwMjg3NTcyNTc4ODQ2Nw=="
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNDk5MzYwMTgzNTU4OTg0ODMxNjA5MzYxODU4NzcxMDczMDI5OTY3NzUzMTk3MzA1Nzg3Mzg4MjQ1MTI5MzAxNTk4MDQ4MDg3ODkxNQ=="
}